src/ast/mod.rs

```diff
@@ -10125,6 +10125,63 @@ impl fmt::Display for MemberOf {
     }
 }
 
+/// Specifies how to create a new table based on an existing table's schema.
+///
+/// Not parenthesized:
+/// '''sql
+/// CREATE TABLE new LIKE old ...
+/// '''
+/// [Snowflake](https://docs.snowflake.com/en/sql-reference/sql/create-table#label-create-table-like)
+/// [BigQuery](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#create_table_like)
+///
+/// Parenthesized:
+/// '''sql
+/// CREATE TABLE new (LIKE old ...)
+/// '''
+/// [Redshift](https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_TABLE_NEW.html)
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub enum CreateTableLikeKind {
+    Parenthesized(CreateTableLike),
+    NotParenthesized(CreateTableLike),
```

Comment:
```suggestion
    Plain(CreateTableLike),
```
Thinking we can do similar to e.g. `CreateTableOptions`, so that `NotParenthesized` doesn't become ambiguous if there happens to be a third variant in the future